kvmagent/kvmagent/plugins/mevoco.py

```
@@ -1170,11 +1231,61 @@ def start(self):
+        try:
+            from kvmagent.plugins import vm_plugin
+
+            event = vm_plugin.LibvirtEventManager.event_to_string(event)
+            if event not in (vm_plugin.LibvirtEventManager.EVENT_STARTED, vm_plugin.LibvirtEventManager.EVENT_STOPPED):
+                return
+
+            if not is_ebtables_nf_tables():
+                return
+
+            bridge_ports = []
+            root = etree.fromstring(dom.XMLDesc(0))
```

Ruff (0.15.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)